database/database.star

```diff
@@ -0,0 +1,16 @@
+postgres = import_module("github.com/kurtosis-tech/postgres-package/main.star")
```

Comment:
is this wrapped up for abstraction? can we use postgres package in main.star?

Reply:
i'd prefer this as the person would be able to see "import" in the main.star

fewer clicks.

Reply:
yes, basically for that, because then we could incorporate MySQL, but probably it's over-engineering. Also at the beginning, I was adding more logic inside, but then I changed the approach, so yes probably it's better to just import it in the main star. I will make the change.

Reply:
done